### DISCUSSION TITLE
Trouble Setting up Item Shadowing in 1.19.2

### DISCUSSION THREAD

**Discussion by swishkin** [fxmorin/carpet-fixes#120] [Q&A] [answered]
Hey, fxmorin!

I've recently migrated from a friend's 1.17 server which I had update suppression, item shadowing, and carpet-shadow working properly on to a 1.19.2 server which I'm also trying to get the same working on. I enabled reIntroduceItemShadowing and reIntroduceInstantBlockUpdates, as directed in [your page on Item Shadowing](https://github.com/fxmorin/carpet-fixes/wiki/How-to-use-item-shadowing-in-1.19-using-carpet-fixes), but the comparator is still popping off when I try to shadow an item.

Prior to reading those directions, I also read that the max-chain-neighbor-updates value in server.properties had to be changed from 1000000 (default) to a negative number for update suppression to work again. However, with it set to -1 (unlimited?), it doesn't work.

Also, I had previously enabled updateSuppressionCrashFix, but it seemed to either have no effect on the update suppression crashes, or _disabling_ the fix _prevented_ the crashes.

Is there a more complete guide to setting a 1.19.* Fabric server up with update suppression and item shadowing working properly?

Here are a few screenshots of the shadowing setup I've got:

![2022-11-27_23 06 33](https://user-images.githubusercontent.com/19404743/204198398-5a1d80af-e67c-482b-9798-e1dc2a1c40e2.png)
![2022-11-27_23 06 38](https://user-images.githubusercontent.com/19404743/204198416-8afcadb2-4eea-4f0d-b2da-ab5452a97fb5.png)
![2022-11-27_23 06 42](https://user-images.githubusercontent.com/19404743/204198431-7fdb64f5-d65a-4ad4-aa11-7162b8b49b53.png)
![2022-11-27_23 07 07](https://user-images.githubusercontent.com/19404743/204198450-1bded523-422d-4c4f-afbf-b456f7cd2809.png)
![2022-11-27_23 07 13](https://user-images.githubusercontent.com/19404743/204198465-a49545b9-ab4c-422d-83a5-44780fe51e6f.png)

Thanks in advance for your reply, and for the carpet-fixes mod!

**Comment by FxMorin** [comment, marked as answer]
Yes it seems it's currently broken and I don't have the time to fix it.

**Reply by swishkin** [reply]
Thanks for your reply!
